src/itemsComponents/aboutItems.js

```diff
@@ -0,0 +1,47 @@
+//assents
```

Comment:
This file need another name like aboutConfig.js and if you create a configuration file for about us pleas make a separate folder for the aboutUs component